### PR TITLE
Documentation 

### PR DESCRIPTION
**Pull request by JacobMGEvans**
- added Author section.
- added prerequisites section.
- Fixed minor grammar mistakes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,8 +26,18 @@ Be sure to install node modules:
...
 ```
 
-It is no ready to start a local development environment:
+It is now ready to start a local development environment:
 
 ```shell
 npm start
 ```
+
+## Authors
```
Comment (kyleshevlin) [inline README.md:35]: I don't feel like an Author, I feel more like a Creator and/or a Founder. I'll need to think about how I want this structured in the readme. I think for now, just cut this section. I'll come back to it.

File: README.md [hunk 2]
```diff
@@ -16,8 +26,18 @@ Be sure to install node modules:
...
+
+- **Kyle Shevlin** - _Initial work_ - [Twitter](https://twitter.com/kyleshevlin)
+
+<!-- See also the list of [contributors](https://github.com/your/project/contributors) who participated in this project. -->
```
Comment (kyleshevlin) [inline README.md:39]: Go ahead and uncomment this, make it:
```
## Contributors

See the list of contributors [here](https://github.com/SecondCareerDevs/website/contributors).
```

We'll make it better with photos and stuff later.

File: README.md [hunk 3]
```diff
@@ -16,8 +26,18 @@ Be sure to install node modules:
...
+<!--
+## License
+
+This project is licensed under the MIT License - see the [LICENSE.md](LICENSE.md) file for details -->
```
Comment (kyleshevlin) [inline README.md:43]: Uncomment this, make it.

```
## License

This project is licensed under the MIT License
```

We can indicate there's a file when the file is added to the repo. FWIW, I don't love having commented out code sitting around the app anywhere, so if you want to add something, my philosophy will be to add what can be added now, make another commit and update later when you have everything.

**Comment by kyleshevlin** [comment]
I'm confused by the merge commit in there. Is there a way for you to get rid of it? I believe you could cherry pick your first commit into a new branch if necessary.

**Comment by JacobMGEvans** [comment]
That happens when I pull down /master from your repo into mine. 

I can probably figure out a way to prevent that from happening. I will be closing this PR and creating a new one later.

**Pull request closed without merging**